tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php

```diff
@@ -1327,6 +1327,245 @@ class Foo {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixPhp74Cases()
+    {
+        return [
+            'some typed static public property' => [
+                '<?php
+class Foo {
+    /**
+     */
+    static public Bar $bar;
+}',
+                '<?php
+class Foo {
+    /**
+     * @var Bar
+     */
+    static public Bar $bar;
+}',
+            ],
+            'some typed public static property' => [
+                '<?php
+class Foo {
+    /**
+     */
+    public static Bar $bar;
+}',
+                '<?php
+class Foo {
+    /**
+     * @var Bar
+     */
+    public static Bar $bar;
+}',
+            ],
+            'some typed public property' => [
+                '<?php
+class Foo {
+    /**
+     */
+    public Bar $bar;
+}',
+                '<?php
+class Foo {
+    /**
+     * @var Bar
+     */
+    public Bar $bar;
+}',
+            ],
+            'some typed protected property' => [
+                '<?php
+class Foo {
+    /**
+     */
+    protected Bar $bar;
+}',
+                '<?php
+class Foo {
+    /**
+     * @var Bar
+     */
+    protected Bar $bar;
+}',
+            ],
+            'some typed private property' => [
+                '<?php
+class Foo {
+    /**
+     */
+    private Bar $bar;
+}',
+                '<?php
+class Foo {
+    /**
+     * @var Bar
+     */
+    private Bar $bar;
+}',
+            ],
+            'some typed nullable private property' => [
+                '<?php
+class Foo {
+    /**
+     */
+    private ?Bar $bar;
+}',
+                '<?php
+class Foo {
+    /**
+     * @var null|Bar
+     */
+    private ?Bar $bar;
+}',
+            ],
+            'some typed nullable property with name declared in phpdoc' => [
+                '<?php
+class Foo {
+    /**
+     */
+    private ?Bar $bar;
+}',
+                '<?php
+class Foo {
+    /**
+     * @var null|Bar $bar
+     */
+    private ?Bar $bar;
+}',
+            ],
+            'some array property' => [
+                '<?php
+class Foo {
+    /**
+     */
+    private array $bar;
+}',
+                '<?php
+class Foo {
+    /**
+     * @var array
+     */
+    private array $bar;
+}',
+            ],
+            'some nullable array property' => [
+                '<?php
+class Foo {
+    /**
+     */
+    private ?array $bar;
+}',
+                '<?php
+class Foo {
+    /**
+     * @var array|null
+     */
+    private ?array $bar;
+}',
+            ],
+            'some object property' => [
+                '<?php
+class Foo {
+    /**
+     */
+    private object $bar;
+}',
+                '<?php
+class Foo {
+    /**
+     * @var object
+     */
+    private object $bar;
+}',
+            ],
+            'phpdoc does not match property typehint' => [
+                '<?php
+class Foo {
+    /**
+     * @var FooImplementation1|FooImplementation2
+     */
+    private FooInterface $bar;
+}',
+            ],
+            'allow_mixed=>false' => [
```

Comment:
The tests without actual type declaration should run on PHP lower than 7.4.

Reply:
Added.